appserver/security/webintegration/pom.xml

```diff
@@ -118,6 +118,10 @@
             <groupId>org.glassfish.gmbal</groupId>
             <artifactId>gmbal</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.omnifaces</groupId>
```

Comment:
After Eclipse Eleos publishes an artifact, should be changed to Eclipse artifact.